src/charts/topology/examples/topology-view.js

```diff
@@ -0,0 +1,314 @@
+        }
+			},
+    });
+
+    $rootScope.data = datasets[index];
+
+    var nodeKinds = {
+			"ContainerReplicator": true,
+			"ContainerGroup": true,
+			"Container": true,
+			"ContainerNode": true,
+			"ContainerService": true,
+			"Host": true,
+			"Vm": true,
+			"ContainerRoute": true,
+			"ContainerManager": true
+		};
+
+		$rootScope.kinds = nodeKinds;
+
+		var icons = $rootScope.data.icons;
+    $scope.nodes = {};
+		for(var kind in nodeKinds) {
+		  var icon = icons[kind];
+		  $scope.nodes[kind] = {
+		    "name": kind,
+		    "enabled": nodeKinds[kind],
+		    "radius": 16,
+		    "textX": 0,
+		    "textY": 5,
+		    "height": 18,
+		    "width": 18,
+		    "icon": icon.icon,
+		    "fontFamily": icon.fontfamily
+		  };
+		}
+
+    // Individual values can also be set for specific icons
+    $scope.nodes.ContainerService.textY = 9;
+    $scope.nodes.ContainerService.textX = -1;
+
+    $scope.nodes.ContainerGroup.height = 30;
+    $scope.nodes.ContainerGroup.width = 30;
+    $scope.nodes.ContainerGroup.radius = 28;
+    $scope.nodes.ContainerGroup.textY = 8;
+
+    $scope.itemSelected = function (item) {
+      var text = "";
+      if (item) {
+        text = "Selected: " + item.name;
+      }
+      angular.element(document.getElementById("selected")).text(text);
+    };
+
+    $scope.removeKind = function () {
+      if($rootScope.kinds.ContainerNode) {
+        delete $rootScope.kinds.ContainerNode;
+      }
+    };
+
+    $scope.tooltip = function (node) {
+      var status = [
+        'Name: ' + node.item.name,
+        'Type: ' + node.item.kind,
+        'Status: ' + node.item.status
+      ];
+      return status;
+    }
+ });
+ </file>
+ <file name="topology.css">
+
+ .pf-topology-svg g.Pod text {
+    font-family: FontAwesome;
+    font-size: 16px;
+    fill: #1186C1;
+  }
+
+   .pf-topology-svg g.Node text {
+    fill: #636363;
+  }
+
+   .pf-topology-svg g.Service text {
+    fill: #ff7f0e;
+  }
+
+   .pf-topology-svg g.ReplicationController text {
+    fill: #9467bd;
+    font-size: 20px;
+  }
+
+   .pf-topology-svg line.ReplicationControllerPod {
+    stroke-linecap: round;
+    stroke-dasharray: 5, 2;
+  }
+
+
+  .pf-topology-svg line.ContainerServiceContainerGroup, .pf-topology-svg line.ContainerReplicatorContainerGroup, .pf-topology-svg line.ContainerServiceContainerRoute,
+   .pf-topology-svg line.ContainerGroupContainerService, .pf-topology-svg line.ContainerGroupContainerReplicator {
+    stroke-linecap: round;
+    stroke-dasharray: 5.5;
+  }
+
+
+ .pf-topology-svg g.Container text.glyph {
+    font-size: 18px;
+  }
+
+   .pf-topology-svg g.ContainerGroup text.glyph {
+    font-size: 28px;
+  }
+
+   .pf-topology-svg g.Vm text.glyph, .pf-topology-svg g.Host text.glyph {
+    fill: #636363;
+  }
+
+   .pf-topology-svg g.ContainerNode text.glyph {
+    font-size: 18px;
+  }
+
+   .pf-topology-svg g.ContainerManager text.glyph {
+    font-size: 18px;
+  }
+ </file>
+
```

Comment:
Yes, due to the fact that they are text icons - this is the way they'll need to be styled.